src/frontend/parser.rs

```diff
@@ -734,6 +734,7 @@ impl FilamentParser {
                     imports: imps,
                     externs: vec![],
                     components: vec![],
+                    toplevel: "".to_string(),
```

Comment:
Should this be main?

Reply:
Yes I think it should, thank you for catching that!